framework/include/media/media.h

```diff
@@ -0,0 +1,55 @@
+#ifndef __MEDIA_H__
+#define __MEDIA_H__
+
+#include <sys/types.h>
+
+/***
+ * Errors
+ */
+#define MEDIA_OK					0
+#define MEDIA_ERROR					1
+
+/***
+ * Enumerations
+ */
+typedef enum {
```

Comment:
Let's split typedef and definition of enum.
There are two reasons.
1. coding rule R11. https://wiki.tizen.org/Native_Platform_Coding_Idiom_and_Style_Guide#C_Coding_Style
2. on doxygen, this causes wrong information.